CGCharter-1727386911.html

```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
+        any action, the seat will automatically be up for election in the next
+        cycle. Any such interim appointments or elections shall hold the seat
+        until the end of its natural cycle.
+      </li>
+      <li>
+        Reelection is restricted to two consecutive terms, with the possibility
+        of being reelected after sitting out one election cycle.
+      </li>
+      <li>
+        In an election year, current chairs will select a date for elections,
+        which will set a nomination period of two weeks, starting 4 weeks prior
+        to the election.
+      </li>
+      <li>
+        For an individual to run for election, they must self-nominate and make
+        a statement regarding their background and why they are running, on the
+        group mailing list.
+      </li>
+      <li>
+        The current chairs will host a conference call during the nomination
+        period, during which candidates may make a statement and answer
+        questions from the community.
+      </li>
+      <li>
+        If, at the end of nominations, any given seat only has a single
+        candidate, that candidate immediately wins that seat. For any seats with
+        multiple nominees, there will be an election for those seats.
+      </li>
+      <li>
+        If, after nominations, any given seat has no candidates, the remaining
+        chairs after any election (if necessary for other seats), will address
+        the vacancy as an interim vacancy, described above.
+      </li>
+      <li>
+        To elect one of multiple candidates, a vote will be held by the election
+        mechanism of ranked choice voting, in which voters rank candidates by
+        preference on their ballots. The candidate with the majority (more than
+        50%) of first-choice votes wins outright. If no candidate gets a
+        majority of first-choice votes, the candidate who ranked the worst is
+        eliminated, and that candidate&quot;s voters&quot; ballots are
+        redistributed to their second-choice pick. If the vote results in a tie,
+        an immediate runoff of the top two candidates shall be held. If the vote
+        remains tied, the winner shall be the candidate whose nomination was
+        first recorded publicly on the group email list.
+      </li>
+    </ul>
+    
+    <h2 id="offboarding-chairs">
+      Chair Offboarding
+    </h2>
+    
+    <ul>
+      <li>
+        A chair may voluntarily off-board at any given time; this should be done
+        publicly and on the mailing list.
+      </li>
+      <li>
+        Chairs may also be removed from their duties through a no-confidence
+        vote.
+      </li>
+      <li>          
+        If a participant of the community group wishes to call for the recall of
+        a chair, for any reason, that participant must first privately
+        communicate with one or more of the other chairs and/or W3C contact
```

Comment:
```suggestion
        communicate with one or more of the other chairs and/or W3C Team Contact
```

Suggestion: make this explicit, it's not just a general contact but *the* Team Contact, right?

Reply:
W3C Team Contact is indeed more specific than a general W3C contact. However, unlike W3C Groups, CGs do not have an assigned W3C Team Contact in the same way. Similarly, CG charters are not formally recognised or enforced by W3C. There is no specific "contact" assigned unless a special arrangement is made between a CG and the W3C Team to designate one.

That said, CGs primarily communicate with the W3C Team's Community Process team at [team-community-process@w3.org](mailto:team-community-process@w3.org) for general admin or process-related matters and potentially for initial Code of Conduct issues. This doesn't extend to CG charters or other CG-specific processes.

So:

```suggestion
        communicate with one or more of the other chairs
```

Aside: The Solid CG charter mentions "W3C Team Contact" in one place but that is technically inaccurate/inapplicable.